dali/dfuplus/main.cpp

```diff
@@ -107,6 +107,7 @@ void handleSyntax()
     out.append("        diffkeysrc=<old-key-name>   -- use keydiff/keypatch (src old name)\n");
     out.append("        diffkeydst=<old-key-name>   -- use keydiff/keypatch (dst old name)\n");
     out.append("        multicopy=0|1   -- each destination part gets whole file\n");
+    out.append("        preservecompression=0|1 -- optional, default is 0 (don't preserve compression)\n");
```

Comment:
From Gavin: 
"If we are preserving backward compatibility, the cleanest way I can think of achieving that is to add a new boolean option to the spray function "preserveCompression" which defaults to false."